aten/src/ATen/native/mkldnn/Matmul.cpp

```diff
@@ -53,8 +53,25 @@ bool mkldnn_fp16_gemm(
     c10::Half *c, int64_t ldc) {
   return false;
 }
+bool mkldnn_bf32_gemm(
```

Comment:
Suggest to add a note somewhere in the code to explain what "bf32" mean.

Reply:
Added.